src/Illuminate/Console/OutputStyle.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Illuminate\Console;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class OutputStyle extends SymfonyStyle
+{
+    /** @var OutputInterface */
+    private $output;
+
+    /**
+     * Create a new Console OutputStyle instance.
+     *
+     * @param  InputInterface $input
+     * @param  OutputInterface $output
+     * @return void
+     */
+    public function __construct(InputInterface $input, OutputInterface $output)
+    {
+        $this->output = $output;
+
+        parent::__construct($input, $output);
+    }
+
+    /**
+     * Returns whether verbosity is quiet (-q)
+     *
+     * @return bool true if verbosity is set to VERBOSITY_QUIET, false otherwise
```

Comment:
Laravel doesn't use return type descriptions like this. Please move this to a method level "long description".